Document On-Call supported countries (#37953)
DataDog/documentation: content/en/incident_response/on-call/notification_preferences/supported_countries.md

---

> title: Supported Countries

Reviewer: ```suggestion
title: Supported Countries
description: A list of countries supported for SMS and phone call notifications in Datadog On-Call, including their country codes.
disable_toc: false
```

further_reading:
- link: '/incident_response/on-call/notification_preferences/'
  tag: 'Documentation'
  text: 'Notification Preferences'
- link: '/incident_response/on-call/'
  tag: 'Documentation'
  text: 'Datadog On-Call'
---

Datadog On-Call supports multiple notification channels for your contact methods: push notifications, email, SMS, and phone calls. See [Supported contact methods][1] to learn more.

> Support for SMS and phone call notifications varies by country because of local regulations and restrictions in that country. Use the following table to check whether your country is supported before adding a phone contact method to your On-Call profile.

Reviewer: ```suggestion
Support for SMS and phone call notifications varies by country because of local regulations and restrictions. Use the following table to check whether your country is supported before adding a phone contact method to your On-Call profile.
```

| India | +91 |
| Indonesia | +62 |
| Ireland | +353 |
| Israel | +972 |
| Italy | +39 |
| Japan | +81 |
| Jersey | +44 |
| Jordan | +962 |
| Kazakhstan | +7 |
| Kenya | +254 |
| Kosovo | +383 |
| Latvia | +371 |
| Lebanon | +961 |
| Liechtenstein | +423 |
| Lithuania | +370 |
| Luxembourg | +352 |
| Malaysia** | +60 |
| Malta | +356 |
| Mexico | +52 |
| Moldova | +373 |
| Monaco | +377 |
| Montenegro | +382 |
| Morocco | +212 |
| Netherlands | +31 |
| New Zealand | +64 |
| Nigeria*, ** | +234 |
| North Macedonia | +389 |
| Norway | +47 |
| Pakistan | +92 |
| Palestine | +970 |
| Peru | +51 |
| Philippines | +63 |
| Poland | +48 |
| Portugal | +351 |
| Puerto Rico | +1 |
| Qatar | +974 |
| Romania | +40 |
| San Marino | +378 |
| Saudi Arabia | +966 |
| Serbia | +381 |
| Singapore | +65 |
| Slovakia | +421 |
| Slovenia | +386 |
| South Africa | +27 |
| South Korea | +82 |
| Spain | +34 |
| Sri Lanka | +94 |
| Svalbard and Jan Mayen | +47 |
| Sweden | +46 |
| Switzerland | +41 |
| Thailand | +66 |
| Tunisia** | +216 |
| Turkey | +90 |
| U.S. Virgin Islands | +1 |
| Ukraine | +380 |
| United Arab Emirates | +971 |
| United Kingdom | +44 |
| United States | +1 |
| Uruguay* | +598 |

> | Åland Islands | +358 |

Reviewer: Minor nit, would customers expect to find it here or after Andorra?

Author: Fair pointed, changed it!
Thanks

---
title: Supported Countries
description: A list of countries supported for SMS and phone call notifications in Datadog On-Call, including their country codes.
disable_toc: false
further_reading:
- link: '/incident_response/on-call/notification_preferences/'
  tag: 'Documentation'
  text: 'Notification Preferences'
- link: '/incident_response/on-call/'
  tag: 'Documentation'
  text: 'Datadog On-Call'
---

Datadog On-Call supports multiple notification channels for your contact methods: push notifications, email, SMS, and phone calls. See [Supported contact methods][1] to learn more.

Support for SMS and phone call notifications varies by country because of local regulations and restrictions. Use the following table to check whether your country is supported before adding a phone contact method to your On-Call profile.

<div class="alert alert-info">
If your country isn't listed in the following table, contact <a href="https://docs.datadoghq.com/help/">Datadog Support</a> to request phone number support for that country.

> </div>

Reviewer 2: ```suggestion
</div>

<div class="alert alert-info">
For the best experience, we recommend downloading the Datadog <a href="https://docs.datadoghq.com/mobile">Mobile App</a> and using push notifications as your primary method of getting paged. SMS and phone call notifications are also available, but local carrier restrictions and regional regulations can occasionally affect their reliability.
</div>
```

| Ireland | +353 |
| Israel | +972 |
| Italy | +39 |
| Japan | +81 |
| Jersey | +44 |
| Jordan | +962 |
| Kazakhstan | +7 |
| Kenya | +254 |
| Kosovo | +383 |
| Latvia | +371 |
| Lebanon | +961 |
| Liechtenstein | +423 |
| Lithuania | +370 |
| Luxembourg | +352 |
| Malaysia** | +60 |
| Malta | +356 |
| Mexico | +52 |
| Moldova | +373 |
| Monaco | +377 |
| Montenegro | +382 |
| Morocco | +212 |
| Netherlands | +31 |
| New Zealand | +64 |
| Nigeria*, ** | +234 |
| North Macedonia | +389 |
| Norway | +47 |
| Pakistan** | +92 |
| Palestine | +970 |
| Peru | +51 |
| Philippines | +63 |
| Poland | +48 |
| Portugal | +351 |
| Puerto Rico | +1 |
| Qatar | +974 |
| Romania | +40 |
| San Marino | +378 |
| Saudi Arabia | +966 |
| Serbia | +381 |
| Singapore | +65 |
| Slovakia** | +421 |
| Slovenia | +386 |
| South Africa | +27 |
| South Korea | +82 |
| Spain | +34 |
| Sri Lanka | +94 |
| Svalbard and Jan Mayen | +47 |
| Sweden | +46 |
| Switzerland | +41 |
| Taiwan | +886 |
| Thailand | +66 |
| Tunisia** | +216 |
| Turkey | +90 |
| U.S. Virgin Islands | +1 |
| Ukraine | +380 |
| United Arab Emirates | +971 |
| United Kingdom | +44 |
| United States | +1 |
| Uruguay* | +598 |

> \* Partial support: Your ability to receive or respond to notifications in these countries might be limited. Datadog sends notification requests to its provider partners in these countries, but local carrier restrictions or government regulations can limit delivery. For these regions, set up push notifications as your primary notification method.

Reviewer 2: Remove the last sentence and instead mention at the top that users should download the mobile app. Push notifications should always be the preferred route, not just in case of partial telecom support.

```suggestion
\* Partial support: Your ability to receive or respond to notifications in these countries might be limited. Datadog sends notification requests to its provider partners in these countries, but local carrier restrictions or government regulations can limit delivery.
```

> \* Partial support: Your ability to receive or respond to notifications in these countries might be limited. Datadog sends notification requests to its provider partners in these countries, but local SMS carrier restrictions or government regulations can limit delivery. For these regions, set up push notifications as your primary notification method.

Author: ```suggestion
\* Partial support: Your ability to receive or respond to notifications in these countries might be limited. Datadog sends notification requests to its provider partners in these countries, but local carrier restrictions or government regulations can limit delivery. For these regions, set up push notifications as your primary notification method.
```

> \*\* Available by request: SMS and phone call notifications for these countries are not enabled by default. Contact <a href="https://docs.datadoghq.com/help/">Datadog Support</a> to request access.

Reviewer 2: ```suggestion
\*\* Available by request: SMS and phone call notifications for these countries are supported but not enabled by default. Contact <a href="https://docs.datadoghq.com/help/">Datadog Support</a> to request access.
```

---
title: Supported Countries
description: A list of countries supported for SMS and phone call notifications in Datadog On-Call, including their country codes.
disable_toc: false
further_reading:
- link: '/incident_response/on-call/notification_preferences/'
  tag: 'Documentation'
  text: 'Notification Preferences'
- link: '/incident_response/on-call/'
  tag: 'Documentation'
  text: 'Datadog On-Call'
---

Datadog On-Call supports multiple notification channels for your contact methods: push notifications, email, SMS, and phone calls. See [Supported contact methods][1] to learn more.

Support for SMS and phone call notifications varies by country because of local regulations and restrictions. Use the following table to check whether your country is supported before adding a phone contact method to your On-Call profile.

<div class="alert alert-info">
If your country isn't listed in the following table, contact <a href="https://docs.datadoghq.com/help/">Datadog Support</a> to request phone number support for that country.
</div>

<div class="alert alert-info">

> For the best experience, we recommend downloading the Datadog <a href="https://docs.datadoghq.com/mobile">Mobile App</a> and using push notifications as your primary method of getting paged. SMS and phone call notifications are also available, but local carrier restrictions and regional regulations can occasionally affect their reliability.

Reviewer: ```suggestion
For the best experience, download the Datadog <a href="https://docs.datadoghq.com/mobile">Mobile App</a> and use push notifications as your primary method of getting paged. SMS and phone call notifications are also available, but local carrier restrictions and regional regulations can affect their reliability.
```
